Makefile

```diff
@@ -61,7 +61,7 @@ HELM_VALUES_TEMPLATE_SKIPPED = true
 # Due to the way that the shared build logic works, images should
 # all be in folders at the same level (no additional levels of nesting).
 
-REGISTRY_ORGS = docker.io/crossplane xpkg.upbound.io/crossplane
+REGISTRY_ORGS ?= docker.io/crossplane xpkg.upbound.io/crossplane
```

Comment:
how is this change related to the functionality implemented in this PR?

Reply:
Not related, indeed. 
But it's too small and noncontroversial to be a separate PR, I assume. 

I did that to allow overriding `REGISTRY_ORGS` from my terminal so that I can build/publish a multi-arch image with my changes using:

```shell
export REGISTRY_ORGS="docker.io/turkenh"
make build.all publish BRANCH_NAME=main
```

The test image in the PR description built like that.

Reply:
Ok, sounds good: https://github.com/crossplane/crossplane/pull/4589